lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEFYarn.cs

```diff
@@ -115,6 +121,39 @@ private void LogApplicationReport()
         }
 
         /// <summary>
+        /// Get application report and log for given appId
+        /// </summary>
+        private void LogApplicationReport(string appId)
+        {
+            Logger.Log(Level.Info, "Getting Application report...");
+            var apps = _reefClient.GetApplicationReports().Result;
+            Logger.Log(Level.Info, "Got Application report: {0}", apps.Count);
+            IApplicationReport report;
+            apps.TryGetValue(appId, out report);
+            if (report != null)
```

Comment:
Right, should directly use if (apps.TryGetValue(appId, out var report))...

What do you mean check logging level? I noticed in Java code, we specify log level in Log(...), we also check if log level is ..., then log. But in C#, we usually don't do it. Because if the current log level is lower than the one in log statement, it won't be logged.